goldstone/core/tests.py

```diff
@@ -333,6 +334,56 @@ def test_predefined_search_func(self):
             self.assertIsInstance(end, datetime.datetime)
 
 
+class AlertSearchModelTests(TestCase):
+    """
+
+    Test the
+    Test SavedSearch model for :
+    1. Given a query-string, build a correct elastic search query object
+    2. Given this query object, execute the query against the log indices
+    3. Return the rows of results back to caller
+    4. Fire an independent elastic search query and compare results with 3.
+    """
+
+    fixtures = ['core_initial_data.yaml']
+
+    def test_loaded_data_from_fixtures(self):
+        self.assertGreater(AlertSearch.objects.all(), 0)
+
+    def test_predefined_alert_func(self):
+
+        saved_alerts = AlertSearch.objects.filter()
+        self.assertIsNotNone(saved_alerts)
+
+        for entry in saved_alerts:
+            search_obj = entry.search()
+            self.assertIsInstance(search_obj, elasticsearch_dsl.search.Search)
+
+        for entry in saved_alerts:
+            search_obj, start, end = entry.search_recent()
+            self.assertIsInstance(search_obj, elasticsearch_dsl.search.Search)
+            self.assertIsInstance(start, datetime.datetime)
+            self.assertIsInstance(end, datetime.datetime)
+            response = search_obj.execute()
+            self.assertGreater(response.hits.total, 0)
+
+            if response.hits.total > 0:
+
+                alert_obj = Alert(name='Test Alert loop :',
```

Comment:
These tests don't test anything meaningful.  When would you get None by creating an Alert object?  When would you get anything other than an object of type Alert?  Same for the EmailProducer tests below.